Concepts.md

```diff
@@ -0,0 +1,141 @@
+# Crosscutting Concepts
+
+![Crosscutting Concepts](./diagrams/08-crosscutting-concepts.drawio.svg)
+
+## Domain concepts
+
+- [WikibaseDataModel](https://github.com/wmde/WikibaseDataModel)
+- [WikibaseDataModelJavaScript](https://github.com/wmde/WikibaseDataModelJavaScript)
+- [WikibaseDataModelTypes](https://github.com/wmde/WikibaseDataModelTypes)
+
+You can read more about Wikibase's data model in the [Glossary](./Glossary.md#entity).
+
+## User Experience concepts (UX)
+
+### User Interface
+
+The only user interface in Wikibase Client is the Data Bridge application which is written in Vue 2 and TypeScript.  
```

Comment:
This paragraph is very client specific, possibly from before moving it to the overview section. It should either be extended with Repo UI concepts or split into Client/Repo sub-sections with a _TBD_ note for Repo.

Reply:
Indeed. I reworked it a bit to make it sound general.